app/src/ui/forks/create-fork-dialog.tsx

```diff
@@ -99,13 +99,18 @@ function renderCreateForkDialogContent(
   return (
     <>
       <DialogContent>
-        {`It looks like you don’t have write access to `}
-        <strong>{repository.gitHubRepository.fullName}</strong>
-        {`. Do you want to create a fork of this repository at `}
-        <strong>
-          {`${account.login}/${repository.gitHubRepository.name}`}
-        </strong>
-        {` to continue?`}
+        <p>
+          {`It looks like you don’t have write access to `}
+          <strong>{repository.gitHubRepository.fullName}</strong>
+          {`. If you should have permission, please check with a repository administrator.`}
```

Comment:
```suggestion
          {`. If you should, please check with a repository administrator.`}
```
@billygriffin maybe we could shorten this a little?